deploy-keycloak.yml

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
Perhaps we should run the _**common**_ role for the key-cloak set up as well like we do for the other servers to have all instances in the same timezone